src/services/asset-inventory/cloud-service/cloud-service-detail/modules/cloud-service-history-detail/CloudServiceHistoryLogTab.vue

```diff
@@ -35,96 +40,179 @@
-                    resource_type: 'value',
+                    name: '2day',
+                    label: i18n.t('LAST_2_DAYS'),
+                    start: dayjs.utc(props.date).subtract(2, 'day'),
+                    end: dayjs.utc(props.date),
                 },
-            ],
+            ])),
+            selectedTimeWithin: 'last6hrs',
+            layouts: [] as DynamicLayout[],
+            dataSourceIds: {} as { [key: string]: string },
+            data: [],
         });
         const handleChangeTab = (tab) => {
             state.activeTab = tab;
         };
+        const getLogData = async () => {
+            try {
+                state.loading = true;
+                const selectedTimeWithin = state.timeWithinList.find(timeWith => timeWith.name === state.selectedTimeWithin);
+                const { logs } = await SpaceConnector.client.monitoring.log.list({
+                    data_source_id: state.dataSourceIds[state.activeTab],
+                    resource_id: props.cloudServiceId,
+                    keyword: state.searchText,
+                    start: selectedTimeWithin?.start.toISOString(),
+                    end: selectedTimeWithin?.end.toISOString(),
+                });
+                state.data = logs;
+                state.loading = false;
+            } catch (e) {
+                ErrorHandler.handleError(e);
+                state.data = [];
+            }
+        };
+        const getSchema = async () => {
+            try {
+                state.loading = true;
+                const { results, total_count } = await SpaceConnector.client.monitoring.dataSource.list({
+                    monitoring_type: 'LOG',
+                    provider: props.provider,
+                });
+                if (!total_count) return;
+                const layouts:DynamicLayout[] = [];
+                const dataSourceIds = {};
+                state.tabs = (results ?? []).map((dataSource) => {
+                    const layout = dataSource.plugin_info?.metadata?.view?.table?.layout;
+                    layouts.push(layout);
+                    dataSourceIds[layout?.name] = (dataSource.data_source_id);
+                    return { name: layout?.name, label: dataSource.name };
+                });
+                state.activeTab = state.tabs[0]?.name;
+                state.layouts = layouts;
+                state.dataSourceIds = dataSourceIds;
+                await getLogData();
+            } catch (e) {
+                ErrorHandler.handleError(e);
+                state.layouts = [];
+                state.dataSourceIds = {};
+            }
+        };
+
+        // handler
+        const dynamicLayoutListeners: Partial<DynamicLayoutEventListener> = {
+            fetch(options) {
+                state.searchText = options?.searchText ?? '';
+                getLogData();
+            },
+        };
+        // init
+        (async () => {
+            await getSchema();
+        })();
```

Comment:
현재 getSchema() 내에 getLogData()가 숨겨져 있어서, 이렇게 플로우를 명확하게 바꾸는 건 어떨까요?
getLogData()에서는 schema totalCount를 파악해서 없다면 early return해주고요.

```suggestion
        (async () => {
            await getSchema();
            await getLogData();
        })();
```